### PR TITLE
as-hash methods created

### PR DESCRIPTION
**Pull request by EugeneDar**

### CODE REVIEW

**Comment by yegor256** [comment]
@EugeneDar it seems like an overkill somewhere. Why can't we simply rely on `hashCode()` for all types?

File: eo-runtime/src/test/eo/org/eolang/bool-tests.eo [hunk 1]
```diff
@@ -144,3 +144,12 @@
           m.write (m.sub 1)
           stdout (sprintf "%d\n" m)
     TRUE
+
+[] > hash-equals-the-value
+  and. > @
+    eq.
+      TRUE
+      TRUE.as-hash
```
[the remark this answers is not in the document]
Comment (EugeneDar) [reply eo-runtime/src/test/eo/org/eolang/bool-tests.eo:152]: Yes, it's a mistake, I fixed it.

File: eo-runtime/src/main/java/EOorg/EOeolang/EOchar$EOas_hash.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import org.eolang.Data;
+import org.eolang.Param;
+import org.eolang.PhDefault;
+import org.eolang.Phi;
+import org.eolang.XmirObject;
+
+@XmirObject(oname = "char.as-hash")
+public class  EOchar$EOas_hash extends PhDefault {
+
+    public static Long hash(Character v) {
```
Comment (yegor256) [inline eo-runtime/src/main/java/EOorg/EOeolang/EOchar$EOas_hash.java:37]: @EugeneDar dead code?

**Comment by yegor256** [comment]
@EugeneDar see my comment above

File: eo-runtime/src/main/java/EOorg/EOeolang/EOchar$EOas_hash.java (hunk 2 above)
Comment (EugeneDar) [reply eo-runtime/src/main/java/EOorg/EOeolang/EOchar$EOas_hash.java]: removed it

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@EugeneDar did you try to run `mvn clean install -Pqulice` locally?

**Comment by EugeneDar** [comment]
@yegor256 yes, thanks, only ran this in the eo-runtime directory and passed successfully. However, the eo directory gives an error.

**Comment by proxzi** [comment]
@yegor256 i have the same problem
Снимок экрана 2022-05-15 в 22 47 31

**Comment by yegor256** [comment]
@EugeneDar @proxzi read the Javadoc of `SnippetTest`

**Comment by EugeneDar** [comment]
@yegor256 I corrected it, and now "mvn clean install -Pqulice" doesn't give an error. Can you please merge it again

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@EugeneDar thanks!

**Pull request merged**